### PR TITLE
Make list interactive

### PR DESCRIPTION
**Pull request by yosefHesham**
**Changes Made**

Added functionalities to make the todo list interactive

---

**Changes Made**

- Create a module to handle task status.
- Clear all finished tasks

### CODE REVIEW

File: modules/change_status.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      checkBox.addEventListener('click', (event) => {
+        if (event.target.checked) {
+          event.target.nextElementSibling.style.textDecoration = 'line-through';
+          event.target.nextElementSibling.style.color = 'gray';
+        } else {
+          event.target.nextElementSibling.style.textDecoration = 'none';
+          event.target.nextElementSibling.style.color = 'black';
+        }
+        ToDoController.changeStatus(Number(event.target.classList[1]));
+        // return;
```
Comment (youmari) [inline modules/change_status.js:16]: - [ ] [OPTIONAL] PLEASE try to remove this unused code

File: modules/change_status.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    checkBox.addEventListener('click', (event) => {
+      if (event.target.checked) {
+        event.target.nextElementSibling.style.textDecoration = 'line-through';
+        event.target.nextElementSibling.style.color = 'gray';
+      } else {
+        event.target.nextElementSibling.style.textDecoration = 'none';
+        event.target.nextElementSibling.style.color = 'black';
+      }
+      ToDoController.changeStatus(Number(event.target.classList[1]));
+      // return;
```
Comment (youmari) [inline modules/change_status.js:34]: - [ ] [OPTIONAL] PLEASE try to remove this unused code

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @yosefHesham 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: modules/change_status.js (hunk 1 above)
Comment (yosefHesham) [reply modules/change_status.js:16]: Thanks very much for the note @youmari

**Pull request merged**